README.md

```diff
@@ -178,6 +188,11 @@ be passed using `[]`, for example: `:token-name[pretty]` would pass the string
 Normally formats are defined using `morgan.format(name, format)`, but for certain
 advanced uses, this compile function is directly available.
 
+The function returned takes arguments `tokens` , `req`, and `res` where `tokens`
+refers to `morgan` itself. If a log should be skipped the function will return
+`null`. The function returned or a custom function can be passed directly to 
```

Comment:
@dougwilson it is about documenting the `format` argument in `morgan(format, options)`. [On line 25 of README.md in this PR](https://github.com/expressjs/morgan/pull/124/files#diff-04c6e90faac2675aa89e2176d2eec7d8R25) @bmeck added the phrase:

> For the syntax of strings and functions see [morgan.compile](#morgancompileformat).

He then added this documentation regarding the semantics of functions returned by `morgan.compile` which is what is linked to by the documentation on line 25.